Dockerfile

```
@@ -1,17 +1,19 @@
 # Build the manager binary
-FROM golang:1.11.4-alpine3.8 as builder
+FROM golang:1.12.10-alpine3.10 as builder
```

```suggestion
FROM golang:1.12-alpine as builder
```
do not hardcode to given go patch and alpine version. We want to have fixed go version together with the newest alpine (with all security fixes etc).